oasis3/.cookies-202311201676916712.txt

```diff
@@ -0,0 +1,6 @@
+# Netscape HTTP Cookie File
+# https://curl.se/docs/http-cookies.html
+# This file was generated by libcurl! Edit at your own risk.
+
+central.xnat.org	FALSE	/	FALSE	0	SESSION_EXPIRATION_TIME	"1676916713468,900000"
+#HttpOnly_central.xnat.org	FALSE	/	FALSE	0	JSESSIONID	central.xnat.org3~4AE18C21AD1208930AFDB7A2285987F7
```

Comment:
Hey! Can you remove all the irrelevant files in the PR? (cookie files and the output log.)